akka-docs/src/main/paradox/serialization.md

```diff
@@ -45,15 +45,17 @@ to disable a default serializer, see @ref:[Disabling the Java Serializer](remoti
 
 ### Enable additional bindings
 
-`akka.Done` is by default serialized by the Java serializer, add the following binding to avoid that.
-It's not enabled by default for compatibility reasons. 
+A few types in Akka are, for backwards-compatibility reasons, still serialized by using Java serializer by default.
+You can switch them add the following binding to avoid that, which will make them serialized using protocol buffers instead.
+Refer to @ref[Rolling Upgrades](#rolling-upgrades) to understand how it is possible to turn and start using these new
+serializers in your clustered applications.
 
 ```
-akka.actor.serialization-bindings {
-  "akka.Done" = akka-misc
-}
+akka.actor.serialization-bindings = ${akka.actor.serialization-bindings} ${akka.actor.java-serialization-disabled-additional-serialization-bindings}
```

Comment:
I think we should keep the more manual config in the docs so that users are in control of what is enabled and not, otherwise we can't add more things to `java-serialization-disabled-additional-serialization-bindings` later.

Reply:
okey 👍